include/libtorrent/settings_pack.hpp

```diff
@@ -2068,6 +2068,16 @@ namespace aux {
 			i2p_inbound_length,
 			i2p_outbound_length,
 
+			// ``announce_port`` is the port passed along to trackers as the
+			// ``&port=`` parameter. If left as the default, the listening port
+			// is used.
```

Comment:
I think you need to mention that 0 means the listen port will not be overridden. I also think it would be nice to give an example of a setup where this would be useful.